packages/cli-platform-android/src/config/findManifest.ts

```diff
@@ -11,7 +11,7 @@ import path from 'path';
 import {unixifyPaths} from '@react-native-community/cli-tools';
 
 export default function findManifest(folder: string) {
-  let manifestPaths = glob.sync(path.join('**', 'AndroidManifest.xml'), {
+  let manifestPaths = glob.sync('**/AndroidManifest.xml', {
```

Comment:
`fast-glob` [only wants paths expressed as forward slashes](https://github.com/mrmlnc/fast-glob#how-to-write-patterns-on-windows), so we want the opposite behaviour.  This was causing failures on the Windows E2E tests.